miniapps/nurbs/nurbs_ex1.cpp

```diff
@@ -10,6 +10,7 @@
 //               nurbs_ex1 -m ../../data/disc-nurbs.mesh -o -1
 //               nurbs_ex1 -m ../../data/pipe-nurbs.mesh -o -1
 //               nurbs_ex1 -m ../../data/beam-hex-nurbs.mesh -pm 1 -ps 2
+//               nurbs_ex1 -m ../../data/segment-nurbs.mesh -r 2 -o 2 -lod 3
```

Comment:
Does glvis work on the 1D output if glvis is rebuilt with this branch?

Reply:
Have not tried. I usually use visit for viz. I will check glvis.

Reply:
No unfortunately not. Apparently Glvis visualizes a 1D solution as a 2D solution  by extruding it.
This extrusion currently does not work for NURBS. Simply adding a NURBS FE to the extrusion  routine (around line 12350 in mesh.cpp) does not work, as NURBS spaces are constructed differently when compared to most other FEs. Therefore it is non-trivial to make the extrusion member functions of the Mesh class work for NURBS.

Reply:
This example outputs glvis and VisIt files, without any option to disable the output. Users may be confused that those files cannot be viewed. The output of these files could be disabled by a flag (possibly parsed from the user's input), which also verifies that the 1D output is not written.